app/src/lib/git/core.ts

```diff
@@ -182,9 +182,14 @@ function getDescriptionForError(error: DugiteError): string {
     case DugiteError.SSHAuthenticationFailed:
     case DugiteError.SSHPermissionDenied:
     case DugiteError.HTTPSAuthenticationFailed:
-      return `Authentication failed. You may not have permission to access the repository or the repository may have been archived. Open ${
-        __DARWIN__ ? 'preferences' : 'options'
-      } and verify that you're signed in with an account that has permission to access this repository.`
+      return `Authentication failed. Some common reasons include:
+- You are not logged in to your account: see ${
+        __DARWIN__ ? 'Preferences' : 'Options'
```

Comment:
```suggestion
        __DARWIN__ ? 'Preferences.' : 'Options.'
```

Added periods for consistency with the other lines.

Reply:
Is it worth calling out how to explicitly access the menu, i.e. `File` > `Options` > `Accounts`, or `GitHub Desktop` > `Preferences` > `Accounts`? Or maybe just `File` > `Options` or `GitHub Desktop` > `Preferences`? Not sure how much clarity is truly needed here.

Reply:
👍 Will swap to the latter suggestion.